README.md

```diff
@@ -1,112 +1,89 @@
-9wm
-====
+## 9wm
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½, also known as rio.
-It provides a very simple and clean user interface.
-It is click-to-type.
-It uses the X11 font system (which, unfortunately, means no Unicode support).
+9wm is an X11 Window manager. It's inspired on the AT&T Plan 9 WM 8½, also known as [rio]. 
 
-9wm does not provide virtual desktops, customization, key bindings,
-EWMH support, or compositing.
-It does not allocate any colors (if you disable COLOR),
-which will be great news if you are stuck in 1993.
+It has four major features:
```

Comment:
```suggestion
```
I'm not sure I would describe all the below as features, but I think we should sidestep the issue and just have a list without a heading.